### ISSUE TITLE
Trying to build a Siamese Network with your code as reference

### ISSUE THREAD

**Issue by problemsniper** [delijati/pytorch-siamese#2]
Hello @delijati 
I am trying to build a siamese network very similar to what you have done, I am looking to feed the outputs from the siamese network arms to a fully connected layer which has a loss function of its own.
Do you know how to do it in pytorch? Any kind of help is much appreciated! 

Thanks
